docs/developer-guide/working-with-custom-formatters.md

```diff
@@ -235,6 +264,76 @@ warning no-unused-vars
   src/configs/clean.js:3:6
 ```
 
+### Using Rule metadata
+
+A formatter that summarizes the rules that were triggered might look like this:
```

Comment:
I realize this is just an example, but summarizing the rules that were triggered seems like a strange thing to use a formatter for, and someone reading the docs might end up misunderstanding why rule metadata is useful. Would it work to add rule metadata to another example instead (e.g. adding rule URLs to one of the existing examples)?

Reply:
Good point. I was trying to avoid cluttering up the "Detailed formatter" example but I think adding just the URL would be fine.